### DISCUSSION TITLE
Power DNS Master Slave replication

### DISCUSSION THREAD

**Discussion by tural183** [PowerDNS/pdns#14299] [General]
Hello team,

Hope you are well

I have question about master / slave replication. 

I created PowerDNS with master slave configuration. But when I enabled replication It is not sending new zone to slave. 
When I created manually domain in slave it is automatically sending records. How can I fix it  and sent it automatically zones. Please help

**Comment by omoerbeek** [comment]
Check autoprimary: https://docs.powerdns.com/authoritative/modes-of-operation.html#autoprimary-automatic-provisioning-of-secondaries or the modern way: https://docs.powerdns.com/authoritative/catalog.html
